chall04/ksappi.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/python3
+
+# Version: python3.7
+
+import sys
+import re
+import hashlib
+
+def calculateHashOfArray(array):
+	arrayStr = ''
+	for row in array:
+		arrayStr += row
+	return hashlib.md5(arrayStr.encode('utf-8')).hexdigest()
+
+try:
+	size = int(input())
+except:
+	print('Give the size of the board as an integer')
+	sys.exit()
```

Comment:
you can use sys.exit("message") ; https://docs.python.org/2/library/sys.html#sys.exit